x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/documents_detail_logic.test.ts

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
[nit] FYI - this file is named `documents_detail` (with an extra s) vs `document_detail` - I think it should probably be the latter, right?

Reply:
273a24b0a12a7a3175e6d54de3570866f63ddda4